lib/ingram_micro/elements/outbound_purchase_order_information.rb

```diff
@@ -6,14 +6,7 @@ class IngramMicro::OutboundPurchaseOrderInformation < IngramMicro::BaseElement
     purchase_order_event: nil,
     currency_code: nil,
     comments: nil
-  }
-
-  def initialize(options = {})
```

Comment:
I changed BaseElement, from which this inherits, to not automatically assign a `customer_id` to each element.